backend/tests/views/dataset.py

```diff
@@ -1,5 +1,6 @@
 """Dataset DRF-Viewset test file"""
-
+import json
+from cmath import log
```

Comment:
Is this used ? I'm surprised I'm not seeing it but I'm surprised Pylint is not throwing a warning about it :thinking: